README.md

```diff
@@ -132,7 +132,6 @@ The following parameters are supported:
 * **http.port:** The port the API  is available at.
 * **http.host:** The hostname under which the Cyface Data Collector is running. This can be something like `localhost`.
 * **http.endpoint:** The path to the endpoint the Cyface Data Collector is running. This can be something like `/api/v3`.
-* **http.port.management:** The port the management API is available at.
```

Comment:
duplicate entry.

Reply:
Na. This is no duplicate entry. The management API runs on a different port. 
However we probably do not need the management API anymore, since it was only responsible for creating users.